pkg/cmd/operator/cmd.go

```diff
@@ -41,6 +42,7 @@ func NewOperatorCommand(streams genericclioptions.IOStreams) *cobra.Command {
 	cmd.AddCommand(NewCleanupJobCmd(streams))
 	cmd.AddCommand(NewGatherCmd(streams))
 	cmd.AddCommand(NewMustGatherCmd(streams))
+	cmd.AddCommand(serveprobes.NewServeProbesCmd(streams))
```

Comment:
serveprobes is a separate package so I wouldn't put it in the command name and do serverprobes.NewCmd instead

Reply:
changed the names (+ there are 2 commands inside that package)